src/blosc2/schunk.py

```diff
@@ -426,6 +426,21 @@ def fill_special(self, nitems, special_value):
         RunTimeError
             If the SChunk could not be filled with the special value.
 
+        Examples
+        --------
+        >>> import blosc2
+        >>> schunk = blosc2.SChunk()
+        >>> # Fill the SChunk with the special value
+        >>> nitems = 200 * 1000
+        >>> print(f"Initial number of chunks: {len(schunk)}")
```

Comment:
`len(schunk)` does not return the number of chunks, but *elements*.  Also, a slightly shorter version would be `f"Initial number of elements: {len(schunk)}"`

Reply:
Although, as this does not bring significant context, I think it would be better to avoid the line entirely.